package.json

```diff
@@ -42,7 +42,7 @@
     "@redux-devtools/app": "^2.1.4",
     "@redux-devtools/extension": "^3.2.2",
     "@reduxjs/toolkit": "^1.8.4",
-    "@replayio/protocol": "0.36.0",
+    "@replayio/protocol": "0.37.0",
```

Comment:
I think this change should have an accompanying change in `yarn.lock`. Did you run `yarn install` ?